modules/datastore/src/Service/ImportService.php

```diff
@@ -202,11 +202,15 @@ private function getNonRecordingParser(string $delimiter) : Csv {
       'quote' => '"',
       'escape' => "\\",
       'record_end' => ["\n", "\r"],
+      'trailing_delimiter' => FALSE,
     ];
 
     $parserConfiguration = $this->dispatchEvent(self::EVENT_CONFIGURE_PARSER, $parserConfiguration);
 
     $parser = Csv::getParser($parserConfiguration['delimiter'], $parserConfiguration['quote'], $parserConfiguration['escape'], $parserConfiguration['record_end']);
+    if (!empty($parserConfiguration['trailing_delimiter'])) {
```

Comment:
Might it not be a little cleaner to expect this to be a bool and therefore have a tighter conditional for it?

```suggestion
    if ($parserConfiguration['trailing_delimiter'] === TRUE) {
```